spec/lib/myfinance/client_spec.rb

```diff
@@ -32,4 +32,11 @@
       subject.entities
     end
   end
+
+  describe '#payable_accounts' do
+    it 'instantiates a new Myfinance::Resources::Entity' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.